crypto/x509/by_dir.c

```diff
@@ -7,12 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
+#include "internal/cryptlib.h"
+#include "e_os.h"
```

Comment:
Flip these two. I mean "e_os.h" should be first.

Reply:
Yes, e_os.h should be first.  Will fix.